api-spec/STAC-standalone.yaml

```diff
@@ -19,8 +19,39 @@ servers:
     description: Production server
 paths:
   /search/stac:
+    get:
+      summary: >-
+        retrieve items matching filters. Intended as a shorthand API for simple queries.

```

Comment:
I'd throw a little '(optional)' bit in here so implementors know it's not required.

Reply:
Looks good - I'll add this to WFS3core+STAC.yaml also.